### DISCUSSION TITLE
Welcome To Wrangler Action Rewrite & Opening of Discussions 🥳

### DISCUSSION THREAD

**Discussion by JacobMGEvans** [cloudflare/wrangler-action#111] [Announcements]
As part of the rewrite/overhaul work to TypeScript and Node runner, we are also opening up discussions for this Repo. 

- The Wrangler Action project on GitHub is being modernized due to lack of maintenance. 
- The project will be rewritten in TypeScript, removing dependencies such as Docker. 
- Docker's removal will decrease spin-up time. 
- Community-requested features, including bulk secrets input, will be added. 
- The project's CI/CD will be fixed and testing added. 
- The command implementation will be improved. 
- Documentation will be updated to reflect changes. 
- The project will begin using Node for the Action engine/runner. 
- All changes will be openly discussed with the community.

Cc: @admah

**Comment by Cherry** [comment]
This is awesome to see! 👀 

@demosjarco has reproduced this action in TypeScript already so that'll likely be a great place to start here: https://github.com/demosjarco/wrangler-action-node

**Reply by demosjarco** [reply]
I would be happy to collaborate on this. Let me know how to help.

**Reply by JacobMGEvans** [reply]
Would love to get a review when it is ready 😄

**Comment by demosjarco** [comment]
If possible, I would like to to uphold the no external dependency idea of coding this from my fork - to only use what's built into NodeJS and GitHub's libraries. That would ensure cross OS compatibility and smaller scripts (faster startup time)

**Reply by JacobMGEvans** [reply]
Other than devDeps, the only dep I really need is `action/core` I was even able to do without `action/exec`

**Reply by demosjarco** [reply]
Yup same. Just `action/core` to communicate the logs and error code status to the nice color and status format of workflow actions
